Form/Type/EmbedType.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Codete\FormGeneratorBundle\Form\Type;
+
+use Codete\FormGeneratorBundle\FormGenerator;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+
+class EmbedType extends AbstractType
+{
+
+    /**
+     * @var FormGenerator $formGenerator
+     */
+    private $formGenerator;
+
+    /**
+     * @param FormGenerator $formGenerator
+     */
+    public function __construct(FormGenerator $formGenerator)
```

Comment:
It's not possible to remove ctor when FormType i registered as Service. 
Also, according to S3.1 docs this way isn't deprecated.